api/views/admin/admin.py

```diff
@@ -26,7 +26,7 @@
 import re
 import shutil
 import stat
-from datetime import datetime
```

Comment:
change the datetime back since we are not using it in here for sessions lifetime anymore